Pipeline/RunIDZCodeReview/RunCodeReview.groovy

```diff
@@ -173,20 +173,22 @@ def createCodeReviewExec(String jobcard, String ruleFile, String customRuleFile,
 //*
 //AKGCREV EXEC PROC=AKGCR
 """
-	steplib.eachWithIndex {it, index ->
+	jcl+="//SYSLIB   DD DUMMY \n"
+	// add identified syslib
+	syslib.eachWithIndex {it, index ->
 		if (index == 0 ) jcl+="//SYSLIB   DD DISP=SHR,DSN=${it} \n"
```

Comment:
We can have 2 SYSLIB   in a JCL ? The second will override the first ?
Maybe we can add //SYSLIB   DD DUMMY only if syslib is empty ?

Reply:
See update